andi/typeutils.py

```diff
@@ -101,3 +103,52 @@ def _get_globalns_for_attrs(func: Callable) -> Dict:
         # (w.r.t. typing.get_type_hints) but will still function
         # correctly.
         return {}
+
+
+def _get_function_types():
+    # from typing module (near get_type_hints), but without ModuleType
+    _function_types = [
+        types.FunctionType,
+        types.BuiltinFunctionType,
```

Comment:
Wonder if `BuiltinMethodType` should also be included here: https://docs.python.org/3/library/types.html#types.BuiltinMethodType

Reply:
Maybe.. But https://github.com/python/cpython/blob/bcc136ba892e62078a67ad0ca0b34072ec9c88aa/Lib/typing.py#L1283 doesn't include it, so likely it is not supported by get_type_hints. Though I can't find an example of what BuiltinMethodType instances.

Reply:
Ok, so better to keep as it is then :+1: